core/src/main/java/hudson/model/Result.java

```diff
@@ -49,42 +54,42 @@
     /**
      * The build had no errors.
      */
-    public static final Result SUCCESS = new Result("SUCCESS",BallColor.BLUE,0,true);
+    public static final @Nonnull Result SUCCESS = new Result("SUCCESS",BallColor.BLUE,0,true);
```

Comment:
This is inconsistent all over the place. I prefer not to touch it unless I have to rewrite the expression significantly.